packages/kolibri-tools/lib/i18n/crowdin.py

```diff
@@ -6,6 +6,7 @@
 import tempfile
 import time
 import zipfile
+from builtins import FileNotFoundError
```

Comment:
I have good news - we no longer support Python 2.7 on develop, so we don't need to do this.

Reply:
Got it! I was wondering why its giving me error. Turns out i was in py2.7 venv 😆